[2016-02-02 17:34:21,781: WARNING/MainProcess] /Users/xianpeng/.virtualenvs/meifenfen/lib/python2.7/site-packages/celery/apps/worker.py:161: CDeprecationWarning: 
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))
[2016-02-02 17:34:21,817: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 2.00 seconds...

[2016-02-02 17:34:23,834: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 4.00 seconds...

[2016-02-02 17:34:27,858: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 6.00 seconds...

[2016-02-02 17:34:33,198: WARNING/MainProcess] Exception
[2016-02-02 17:34:33,199: WARNING/MainProcess] KeyError
[2016-02-02 17:34:33,199: WARNING/MainProcess] :
[2016-02-02 17:34:33,199: WARNING/MainProcess] KeyError(140735158321152,)
[2016-02-02 17:34:33,199: WARNING/MainProcess] in
[2016-02-02 17:34:33,199: WARNING/MainProcess] <module 'threading' from '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.pyc'>
[2016-02-02 17:34:33,199: WARNING/MainProcess] ignored
[2016-02-02 17:34:38,308: WARNING/MainProcess] /Users/xianpeng/.virtualenvs/meifenfen/lib/python2.7/site-packages/celery/apps/worker.py:161: CDeprecationWarning: 
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))
[2016-02-02 17:34:38,342: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 2.00 seconds...

[2016-02-02 17:34:40,356: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 4.00 seconds...

[2016-02-02 17:34:44,371: ERROR/MainProcess] consumer: Cannot connect to amqp://monitor:**@127.0.0.1:5672//: [Errno 61] Connection refused.
Trying again in 6.00 seconds...

[2016-02-02 17:34:47,498: WARNING/MainProcess] Exception
[2016-02-02 17:34:47,498: WARNING/MainProcess] KeyError
[2016-02-02 17:34:47,498: WARNING/MainProcess] :
[2016-02-02 17:34:47,498: WARNING/MainProcess] KeyError(140735158321152,)
[2016-02-02 17:34:47,498: WARNING/MainProcess] in
[2016-02-02 17:34:47,498: WARNING/MainProcess] <module 'threading' from '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.pyc'>
[2016-02-02 17:34:47,498: WARNING/MainProcess] ignored
